gui/src/app/SamplerOutputView/SummaryView.tsx

```diff
@@ -161,6 +161,7 @@ const computeEss = (x: number[], chainIds: number[]) => {
         draws[chainIndex].push(x[i]);
     }
     const ess = compute_effective_sample_size(draws);
+    // const ess = compute_split_effective_sample_size(draws);
```

Comment:
I think this is from a merge gone awry or something

Reply:
Fixed.